runTest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+
+touch output.txt
+
+./a.out Units/Unit1.json Units/Unit2.json >> output.txt
+./a.out Units/Unit1.json Units/Unit3.json >> output.txt
+./a.out Units/Unit2.json Units/Unit1.json >> output.txt
+./a.out Units/Unit2.json Units/Unit3.json >> output.txt
+./a.out Units/Unit3.json Units/Unit1.json >> output.txt
+./a.out Units/Unit3.json Units/Unit2.json >> output.txt
+
+if cmp -s output.txt result.txt
+then
+    exit 0
+else
+    exit 1
+fi
```

Comment:
nem crucial, es nem vagyok bash magus, de gondolom ezt is lehetne akkor egy sorban, nem? :-)

Reply:
Tényleg, távoli kicsit a bash nekünk :(

Reply:
Evente kb 3x irok bash kodot en is, szoval sajnos nekem sincs a kezemben, de akarmilyen nyelven van egy `if feltetel then return true else return false` az szinte biztos leirhato `return feltetel` formaban igy vagy ugy :-)